<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookmarkInfoDialog</class>
 <widget class="QDialog" name="BookmarkInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="name_lbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>62</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="description_lbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="description">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Description&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_saveButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_cancelButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="folder_lbl">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>62</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Folder</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_folders">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>150</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_newFolderButton">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>190</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Folder</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_saveButton</sender>
   <signal>clicked()</signal>
   <receiver>BookmarkInfoDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>BookmarkInfoDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
